go.mod

```diff
@@ -20,15 +20,15 @@ require (
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
 	github.com/aliyun/alibaba-cloud-sdk-go v1.62.3
 	github.com/aws/aws-sdk-go v1.50.8
-	github.com/aws/aws-sdk-go-v2 v1.24.1
-	github.com/aws/aws-sdk-go-v2/config v1.26.6
-	github.com/aws/aws-sdk-go-v2/credentials v1.16.16
-	github.com/aws/aws-sdk-go-v2/feature/s3/manager v1.15.15
-	github.com/aws/aws-sdk-go-v2/service/athena v1.12.0
-	github.com/aws/aws-sdk-go-v2/service/ec2 v1.29.0
-	github.com/aws/aws-sdk-go-v2/service/s3 v1.48.1
-	github.com/aws/aws-sdk-go-v2/service/sts v1.26.7
-	github.com/aws/smithy-go v1.19.0
+	github.com/aws/aws-sdk-go-v2 v1.25.1
```

Comment:
Is this version of `aws-sdk-go-v2` fixed? I recall we had some trouble with a recent version

Reply:
@michaelmdresser If the issue you are referring to is: https://github.com/aws/aws-sdk-go-v2/issues/2370#issuecomment-1816904679

Then yes. The TL;DR is AWS made a breaking change in the middleware in their SDK, and mixing and matching AWS packages before or after 11/15 will not work (which we were doing, and was the root of the issues).